archive/dto.ksy

```diff
@@ -0,0 +1,57 @@
+meta:
+  id: dto
+  title: DTB/DTBO
+  license: CC0-1.0
+  file-extension: img
+  endian: be
+
+doc-ref:
+  - https://source.android.com/devices/architecture/dto/partitions
+  - https://android.googlesource.com/platform/system/libufdt/+/refs/heads/master/utils/
+
+types:
+  header:
+    seq:
+    - id: magic
+      contents: [ 0xd7, 0xb7, 0xab, 0x1e ]
+    - id: total_size
+      type: u4
+    - id: header_size
+      type: u4
+    - id: dt_entry_size
+      type: u4
+    - id: dt_entry_count
+      type: u4
+    - id: dt_entries_offset
+      type: u4
+    - id: page_size
+      type: u4
+    - id: version
+      type: u4
+  entry:
+    seq:
+    - id: size
+      type: u4
+    - id: offset
+      type: u4
```

Comment:
Names of these attributes are inadequate, because they lack any information about which field they relate to. It is recommended to call them `len_body` and `ofs_body`, see the [style guide](https://doc.kaitai.io/ksy_style_guide.html#attr-id).